projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.scss

```diff
@@ -21,6 +21,37 @@
     padding: 32px;
     overflow: scroll;
 
+    &__preview{
```

Comment:
slightly wrong formatting, missing spaces before brackets, and the color of the "p" should be coming from the CSS variable to support theming